### DISCUSSION TITLE
The hyperlink to the MegaDepth is broken

### DISCUSSION THREAD

**Discussion by HUSTNO1WXY** [zju3dv/LoFTR#210] [Q&A]
Link directing to MegaDepth pre-processed by D2Net has been invalid.(Google drive)

**Comment by leon5678** [comment]
hi, did you find the way to download the dataset pre-processed by d2net?

> Link directing to MegaDepth pre-processed by D2Net has been invalid.(Google drive)

**Reply by Penge666** [reply]
Can anyone share the link

**Reply by EdithYoung** [reply]
Did you find the link? It's still invalid now. If you found the link, please share it. Thank you!
